developerportal/apps/home/models.py

```diff
@@ -79,13 +79,26 @@ class HomePage(BasePage):
                     ),
                 ),
                 ("external_page", FeaturedExternalBlock()),
+                (
+                    "video",
+                    PageChooserBlock(
+                        target_model=(
+                            "videos.Video",
+                            # NB: ExternalVideo is NOT allowed on the homepage
+                            # "externalcontent.ExternalVideo"
+                        )
+                    ),
+                ),
```

Comment:
This addition is just configuring a StreamBlock in Wagtail's StreamField to allow links to a new type of content - specifically a `Video` page, but only the type where we host the page containing the video